README.md

```diff
@@ -27,6 +27,7 @@ title: Tutorials
 * [TUTORIAL: How to build a To-do list Æpp - Part 1](build-to-do-list-aepp-1.md)
 * [TUTORIAL: How to build a To-do list Æpp - Part 2](build-to-do-list-aepp-2.md)
 * [TUTORIAL: Forgae has made it simple to build Æpps with the front-end framework](forgae-shape-vue-command.md)
+* [TUTORIAL: How to make wallet transactions on android using aepp-java-sdk](making-aeternity-wallet-transactions-on-android.md)
```

Comment:
**Android** instead of "android" to be consistent with the heading of the tutorial.

Reply:
Ok, have done that now